### DISCUSSION TITLE
Create additional scratch org definitions

### DISCUSSION THREAD

**Discussion by allisonletts** [SFDO-Community-Sprints/Snowfakery-Recipe-Templates#78] [Ideas]
We need to decide how NPSP and EDA orgs, as well as the other boxes, work.
@eehjunggnujhee

**Comment by mjh-jhm** [comment]
@allisonletts  This is fairly easy technically but we will need a nomenclature for CCI orgs on the testing workflow. 

1. Create a CCI org definition in the orgs folder. It can be as simple as cloning the existing org definition JSON and rename the file with an alias for a new CCI org  - for example, npsp.json and eda.json
2. When connecting a CCI org, simply declare the new cci org alias at the end of ‘CCI org connect <orgAlias>’ or on the —org option

**Comment by allisonletts** [comment]
Which cci orgs should we define, is really the question. Since a lot of the original cci documentation refers to the QA org (as in, it gives examples of how to run `cci flow run dev_org --org qa`) anyone who's referencing that documentation is at risk of trying the QA org definition. We don't define a QA org ourselves, and I think we should. It may also help to add specific options for NPSP or EDA orgs, but in my mind QA is the most important.

**Reply by allisonletts** [reply]
Here's what's in the standard CCI YAML file. So qa in fact uses the dev org definition. Red herring. The others are likely more worthwhile. 

![image](https://user-images.githubusercontent.com/17914696/141650539-b32f0e38-c223-46a9-ac23-e81af2315d13.jpeg)

**Reply by mjh-jhm** [reply]
Yap yap! CCI orgs’ naming convention follow the development cycle but we can definitely morph a bit to for our need without too much customization.
